docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -1,5 +1,22 @@
 # This document lists known breaking changes in Roslyn after .NET 6 all the way to .NET 7.
 
+## `scoped` treated as a contextual keyword
```

Comment:
Should this be "Types cannot be named `scoped`" instead, assuming that captures the break, for consistency with the changes for `file` and `required`?